src/modules/navigation/components/Navigation/Navigation.consts.tsx

```diff
@@ -193,7 +193,7 @@ export const getNavigationItemsLeft = (
 		],
 	},
 	{
-		node: renderLink('Admin', '/admin/leeszalenbeheer/aanvragen', {
+		node: renderLink('Admin', '', {
```

Comment:
For admin they asked not to add the dropdown menu during the last demo. Just show the admin link.
```suggestion
		node: renderLink('Admin', '/admin/leeszalenbeheer/leeszalen', {
```

Reply:
Staat dat ergens gedocumenteerd? Ontbreekt precies op https://meemoo.atlassian.net/wiki/spaces/HA2/pages/3448504321/2022-04-14+demo+sprint+9+deel+1+BZT

Reply:
Nee blijkbaar niet. Ik zet het op de PO sync om dat vast te leggen: https://meemoo.atlassian.net/wiki/spaces/HA2/pages/3463512065/2022-04-28+PO+sync